### DISCUSSION TITLE
Form.Item  包裹组件的 value onChange  ts 错误如何处理?

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#45755] [Q&A] [answered]
<img width="1029" alt="image" src="https://github.com/ant-design/ant-design/assets/124666577/99306af3-1061-4419-a9e3-d43b1334d1cd">

这有办法解决吗?

**Comment by zombieJ** [comment]
贴个 codesandbox ，截图连个错误信息都没有。很难给建议的

**Reply by vaynevayne** [reply]
<img width="659" alt="image" src="https://github.com/ant-design/ant-design/assets/124666577/5e7f719c-eef0-4837-ae6c-cc5312e7ba59">

**Reply by zombieJ** [reply]
```diff
- value: xxx;
+ value?: xxx;
```

**Reply by vaynevayne** [reply]
我这组件 value 是必传的,否则内部 ts 报错

**Reply by zombieJ** [reply, marked as answer]
那就设置一下 value={null!}
